WalletWasabi.Gui/Tabs/SettingsViewModel.cs

```diff
@@ -50,6 +52,7 @@ internal class SettingsViewModel : WasabiDocumentTabViewModel
 		public SettingsViewModel(Global global) : base(global, "Settings")
 		{
 			Autocopy = Global.UiConfig?.Autocopy is true;
+			FeeDisplayFormatString = ConvertFeeDisplayFormat(Global.UiConfig?.FeeDisplayFormat ?? 0);
```

Comment:
Is this cargo cult or is it really possible to have a `null``Global.UiConfig` at this point?

Reply:
It yells at me if I don't null check